Makefile

```diff
@@ -4,6 +4,10 @@ SOURCES := src src/common src/desktop
 ARGS    := -Ofast -flto -std=c++11 #-DDEBUG
 LIBS    := -lportaudio
 
+APPNAME      ?= NooDS
+ORGANIZATION ?= com.github.hydr8gon

```

Comment:
I'm also wondering about the necessity of having this in all the filenames. I took a look in my `applications` folder and nothing really seems to follow this, except for some GNOME stuff. And, dunno, it feels weird having this as if it's a configurable thing when it's hardcoded in the filenames.